jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -271,14 +273,22 @@ private void extractTar() {
   }
 
   private void assignLocalImageResult(Future<LocalImage> localImage) {
-    results.baseImageLayers = executorService.submit(() -> localImage.get().layers);
     results.baseImageAndRegistryClient =
         executorService.submit(
             () ->
                 LocalBaseImageSteps.returnImageAndRegistryClientStep(
-                        realizeFutures(results.baseImageLayers.get()),
+                        realizeFutures(localImage.get().layers),
                         localImage.get().configurationTemplate)
                     .call());
+
+    results.baseImagesAndLayers =
+        executorService.submit(
+            () -> {
+              Map<Image, List<Future<PreparedLayer>>> baseImageAndLayers = new HashMap<>();
+              baseImageAndLayers.put(
+                  results.baseImageAndRegistryClient.get().images.get(0), localImage.get().layers);
+              return baseImageAndLayers;
```

Comment:
`Collections.signletonMap` (https://github.com/GoogleContainerTools/jib/pull/2646#discussion_r463274516)?

Reply:
I had missed this review